aptos-move/framework/aptos-framework/sources/Block.move

```diff
@@ -57,9 +57,10 @@ module AptosFramework::Block {
     /// The runtime always runs this before executing the transactions in a block.
     fun block_prologue(
         vm: signer,
+        epoch: u64,
         round: u64,
         timestamp: u64,
-        previous_block_votes: vector<address>,
+        previous_block_votes: vector<bool>,
```

Comment:
proposer is not an address anymore? it's a u64 now

Reply:
Oops! Thanks for catching that!